tasks/request_status_for_job/test/unit_tests/test_request_status_for_job_unit.py

```diff
@@ -0,0 +1,213 @@
+            mock_setMetadata: MagicMock,
+            mock_get_dbconnect_info: MagicMock
+    ):
+        """
+        If job_id is not present, should return an error dictionary.
+        """
+        event = {
+        }
+        context = Mock()
+        result = request_status_for_job.handler(event, context)
+        self.assertEqual(HTTPStatus.BAD_REQUEST, result['httpStatus'])
+
+    # noinspection PyPep8Naming
+    @patch('request_status_for_job.task')
+    @patch('requests_db.get_dbconnect_info')
+    @patch('cumulus_logger.CumulusLogger.setMetadata')
+    def test_handler_database_error_returns_error_code(
+            self,
+            mock_setMetadata: MagicMock,
+            mock_get_dbconnect_info: MagicMock,
+            mock_task: MagicMock
+    ):
+        """
+        If database error is raised, it should be caught and returned in a dictionary.
+        """
+        job_id = uuid.uuid4().__str__()
+
+        event = {
+            request_status_for_job.INPUT_JOB_ID_KEY: job_id
+        }
+        context = Mock()
+        mock_task.side_effect = requests_db.DatabaseError()
+        result = request_status_for_job.handler(event, context)
+        self.assertEqual(HTTPStatus.INTERNAL_SERVER_ERROR, result['httpStatus'])
+
+    @patch('request_status_for_job.get_granule_status_entries_for_job')
+    @patch('request_status_for_job.get_status_totals_for_job')
+    def test_task_happy_path(
+            self,
+            mock_get_status_totals_for_job: MagicMock,
+            mock_get_granule_status_entries_for_job: MagicMock
+    ):
+        """
+        Basic path with all information present.
+        """
+        job_id = uuid.uuid4().__str__()
+        db_connect_info = Mock()
+
+        result = request_status_for_job.task(job_id, db_connect_info)
+
+        mock_get_granule_status_entries_for_job.assert_called_once_with(job_id, db_connect_info)
+        mock_get_status_totals_for_job.assert_called_once_with(job_id, db_connect_info)
+
+        self.assertEqual({
+            request_status_for_job.OUTPUT_JOB_ID_KEY: job_id,
+            request_status_for_job.OUTPUT_JOB_STATUS_TOTALS_KEY: mock_get_status_totals_for_job.return_value,
+            request_status_for_job.OUTPUT_GRANULES_KEY: mock_get_granule_status_entries_for_job.return_value
+        }, result)
+
+    @patch('database.result_to_json')
+    @patch('database.single_query')
+    def test_get_granule_status_entries_for_job_happy_path(
+            self,
+            mock_single_query: MagicMock,
+            mock_result_to_json: MagicMock
+    ):
+        """
+        Basic path with all information present.
+        """
+        job_id = uuid.uuid4().__str__()
+
+        db_connect_info = Mock()
+
+        result = request_status_for_job.get_granule_status_entries_for_job(job_id, db_connect_info)
+
+        mock_single_query.assert_called_once_with(f"""
+            SELECT
+                granule_id as \"{request_status_for_job.OUTPUT_GRANULE_ID_KEY}\",
```

Comment:
Since you are in a doc string I don't believe you need to dereference the the quotes `\"`.